db/migrate/20230118185808_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.string :Text
```

Comment:
- [ ] Same here kindly make sure to use `text` for `Text` instead of `string` so your migrations are created based on the `ERD`

Reply:
Updated. Thank You